cognite/neat/_session/_state.py

```diff
@@ -27,12 +30,36 @@ def input_rule(self) -> ReadRules:
             raise ValueError("No input rules provided")
         return self.input_rules[-1]
 
+    @property
+    def information_input_rule(self) -> ReadRules | None:
+        if self.input_rules:
+            for rule in self.input_rules[::-1]:
+                if isinstance(rule.rules, InformationInputRules):
+                    return rule
+        return None
+
     @property
     def verified_rule(self) -> VerifiedRules:
         if not self.verified_rules:
             raise ValueError("No verified rules provided")
         return self.verified_rules[-1]
 
+    @property
+    def verifies_dms_rules(self) -> DMSRules | None:
+        if self.verified_rules:
+            for rules in self.verified_rules[::-1]:
+                if isinstance(rules, DMSRules):
+                    return rules
+        return None
+
+    @property
+    def verifies_information_rules(self) -> InformationRules | None:
```

Comment:
Just to make it clear for us as developers.
```suggestion
    def last_verifies_information_rules(self) -> InformationRules | None:
```